src/pages/Register.tsx

```diff
@@ -0,0 +1,294 @@
+        backgroundSize: '750px',
+        backgroundPosition: 'right 0px bottom -600px',
+      }}
+    >
+      {registrationLogo()}
+      <div
+        style={{
+          marginTop: '4rem',
+          color: colors.dark(0.6),
+          fontSize: '1.5rem',
+          fontWeight: 500,
+        }}
+      >
+        New User Registration
+      </div>
+      <div style={{ marginTop: '1rem', display: 'flex' }}>
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            required
+            value={givenName}
+            onChange={setGivenName}
+            inputStyle={{ display: 'block' }}
+            label="First Name"
+          />
+        </div>
+        <div style={{ width: '1rem' }} />
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={familyName}
+            onChange={setFamilyName}
+            inputStyle={{ display: 'block' }}
+            label="Last Name"
+          />
+        </div>
+      </div>
+      <div style={{ lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={email}
+          required
+          onChange={setEmail}
+          labelStyle={{ display: 'block', marginTop: '2rem' }}
+          inputStyle={{ width: '66ex' }}
+          label="Contact Email for Notifications"
+        />
+      </div>
+      <div style={{ lineHeight: '170%' }}>
+        <LabelledTextInput
+          value={institute}
+          required={partOfOrganization}
+          disabled={!partOfOrganization}
+          onChange={setInstitute}
+          inputStyle={{ width: '66ex' }}
+          label="Organization"
+        />
+      </div>
+      <div style={{ lineHeight: '170%', marginTop: '0.25rem' }}>
+        <LabeledCheckbox
+          checked={!partOfOrganization}
+          onChange={() => setPartOfOrganization(!partOfOrganization)}
+          disabled={false}
+        >
+          <label style={{ marginLeft: '0.25rem' }}>I am not a part of an organization</label>
+        </LabeledCheckbox>
+      </div>
+      <div style={{ display: 'flex' }}>
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={department}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setDepartment}
+            labelStyle={{ display: 'block' }}
+            label="Department"
+          />
+        </div>
+        <div style={{ width: '1rem' }} />
+        <div style={{ lineHeight: '170%' }}>
+          <LabelledTextInput
+            value={title}
+            required={partOfOrganization}
+            disabled={!partOfOrganization}
+            onChange={setTitle}
+            label="Title"
+            labelStyle={{ display: 'block' }}
+          />
+        </div>
+      </div>
+      <h3 style={{ marginTop: '2rem' }}>I am most interested in using Terra to (Check all that apply):</h3>
+      <CheckboxLine>
+        <InterestInTerraCheckbox
```

Comment:
Is there a syntax to iterate over these different options instead of having to explicitly create the list?

Reply:
I'm sure there is, this was just a first pass in some free time.

Reply:
Implemented it with underscore, could have also done it imperatively with a `for` loop